### PR TITLE
fix: replace dependency `open` with `opn`

### PR DESCRIPTION
**Pull request by sibiraj-s**
This won't change anything, just replaced a devDependency. While installing, npm reported a critical vulnerability warning with [open] and which is no longer maintained hence replaced with [opn]

[open]: https://github.com/jjrdn/node-open
[opn]: https://github.com/sindresorhus/opn

### CODE REVIEW

File: bin/open-config.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env node
 
-const config = require('../src/config')
-const open = require('open')
+const { CONFIG_PATH } = require('../src/config')
+const opn = require('opn')
 
-open(config.CONFIG_PATH)
+opn(CONFIG_PATH).catch((err) => {
+  if (err) {
```
[the remark this answers is not in the document]
Comment (sibiraj-s) [reply bin/open-config.js]: Actually, i didn't add that first. Unlike `open`, `opn` throwed unhandled promise rejection error, If path doesn't exists. If still that is not necessary. I will remove that.

[the remark this answers is not in the document]
Comment (mathiasvr) [reply bin/open-config.js]: I'm talking about removing the if condition, not the catch call.
Like this: `opn().catch(err => console.log())`

[the remark this answers is not in the document]
Comment (sibiraj-s) [reply bin/open-config.js]: 😅didn't see that. I do this err handling in my project which is a little old and it became a habit.

[the remark this answers is not in the document]
Comment (mathiasvr) [reply bin/open-config.js]: No problem, thanks for updating it! 👍

File: bin/open-config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-const config = require('../src/config')
-const open = require('open')
+const { CONFIG_PATH } = require('../src/config')
+const opn = require('opn')
 
-open(config.CONFIG_PATH)
+opn(CONFIG_PATH).catch(() => console.log('Unable to open: ' + CONFIG_PATH))
```
[the remark this answers is not in the document]
Comment (sibiraj-s) [reply bin/open-config.js]: We don't.  But handled for good.

[the remark this answers is not in the document]
Comment (Borewit) [reply bin/open-config.js]: Not likely to be ever thrown you mean?

In that case suppressing the error is a bad thing. I would prefer to [fail fast](https://en.wikipedia.org/wiki/Fail-fast).
Either print what went wrong to console.error, and throw that or another error right after. or just don't catch.

[the remark this answers is not in the document]
Comment (sibiraj-s) [reply bin/open-config.js]: Okay. Will do that 👍

[the remark this answers is not in the document]
Comment (sibiraj-s) [reply bin/open-config.js]: updated.

**Comment by sibiraj-s** [comment]
@Borewit I rebased with master and saw that lock file is restored. Which is pretty old and . It had around 117 vulnerabilities. So I regenerated the lock file too.

**Comment by Borewit** [comment]
Housekeeping of dependencies could be a bit better, sorry that it fall over you.

Thanks for your effort @sibiraj-s!

**Comment by Borewit** [comment]
Please sync with master @sibiraj-s.

**Comment by sibiraj-s** [comment]
done 👍  @Borewit

**Comment by Borewit** [comment]
Thx @sibiraj-s!

**Comment by mathiasvr** [comment]
This PR unfortunately downgraded many dependencies that I updated in #1486.

In the future, please ensure that only a couple of lines change in the package-lock file before merging.

**Comment by Borewit** [comment]
Sorry about that @mathiasvr.

**Comment by sibiraj-s** [comment]
Sure, I didn't do anything actually. Just rebased. `npm install` to resolve conflicts. npm is using local `node_modules` folder to resolve conflicts and keep the best match, which is not doing a great job with conflicts maybe. 😕

**Comment by sibiraj-s** [comment]
@Borewit I recommend to delete the lock file and regenerate it once again in master. For this is to not continue on some existing PR's if it has a lock file commit.

**Comment by mathiasvr** [comment]
@Borewit No problem, it's no big deal.

@sibiraj-s Not your fault really, the transition to using a lock file has been a mess, sorry you got caught in the middle of it.

The fact that `npm install` did not acquire to the dependencies in `package-lock.json` seems problematic.

**Comment by Borewit** [comment]
@mathiasvr Can you create a PR with updated dependencies?

**Pull request merged**